### PR TITLE
fix: robotics bug

### PR DESCRIPTION
**Pull request by Gogetter**
### JIRA link (if applicable) ###
[FPLA-1326](https://tools.hmcts.net/jira/browse/FPLA-1326)

### Change description ###
- Update telephone/number regex reverting min of 10
- Fix NPE

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java [hunk 1]
```diff
@@ -42,6 +42,27 @@ public static RoboticsData expectedRoboticsData(final String applicationType) {
             .build();
     }
 
+    public static RoboticsData expectedRoboticsData(final String applicationType, final String phoneNumber) {
```
Comment (akomisarek) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java]: This introduces a lot of duplication with the method above, probably could invoke the method above and work with the partially prepared core data? What are your thoughts?

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java [hunk 2]
```diff
@@ -156,4 +177,24 @@ public static Applicant expectedApplicant() {
             .build();
     }
 
+    public static Applicant expectedApplicant(final String number) {
```
Comment (akomisarek) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/SampleRoboticsTestDataHelper.java]: Again duplication here. Couldn't you invoke this method from the oone above passing appropriate parameters?

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java [hunk 3]
```diff
@@ -448,11 +465,11 @@ void shouldThrowRoboticsDataExceptionWhenApplicantInternationalMobileNumberIsInV
     }
 
     private static Stream<String> inValidMobileNumbers() {
-        return Stream.of("0777", "078888888888888888888888656");
+        return Stream.of("c/o yo!", "078888888888888888888888656");
     }
 
     private static Stream<String> inValidInternationalMobileNumbers() {
-        return Stream.of("", "+1800801920777777777888886565557778888", "c/o");
+        return Stream.of("+1800801920777777777888886565557778888", "c/o");
```
[the remark this answers is not in the document]
Comment (Gogetter) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java]: yes, this is in `shouldNotThrowRoboticsDataExceptionWhenApplicantInternationalPhoneNumberIsValid` which has a variant of numbers with space/characters (that gets trimmed making the number valid)

[the remark this answers is not in the document]
Comment (akomisarek) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java]: Ok but we rely on the validation only, we don't really check the output, maybe we should? 🤔

[the remark this answers is not in the document]
Comment (Gogetter) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/robotics/RoboticsDataServiceTest.java]: we should. we include 👍

**Pull request merged**